### DISCUSSION TITLE
Внедрить отображение версии/номера сборки

### DISCUSSION THREAD

**Discussion by dsamsonov1** [MarchiyGV/RotationAndDeposition#8] [Ideas]
Предлагаю ввести отображение где-либо номера версии, потому что иначе не ясно, как привязывать обсуждения/багрепорты и пр.

**Comment by MarchiyGV** [comment]
где привязать, в программе?

**Comment by dsamsonov1** [comment]
Да, наверняка питон умеет как-то внедрять в сборку номер коммита или обновлять номер сборки. И этот номер надо где-то писать в программе. Обычно это делают в окне Help->About
Вот пример:
![image](https://user-images.githubusercontent.com/61458338/179837784-7573c1d0-4185-4db0-87cf-14a3e935f33f.png)
